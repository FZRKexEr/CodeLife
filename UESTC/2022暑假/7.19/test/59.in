10
ephyfuaqtqijeggesiuysptcyqatrvfxgffsocuchbdaihwlrzobxpzxjgtaotkhitzyidkxmccwlstchfhvvwsvvcrehdcdkggw
4
bkdpsciymojjazxxnshujnkoksupadhnxtqzujdcrkzzbbgrwlgzpboobpdsxwmsbjeqcvmombypemfrncxyxccvnildabgusvcl
1
jdforxdxjbtnipxcsvumevesdlzqbsfhzapmxapkfpvcjmmmncwymweezroydnybdbpfygwobznwzvgxahmivlnpshzkomcloims
9
bdmugvzqrdgtkfakicyvjwxqpycmrbfyqdcjhpjglmrhxdvancrzndumcwcpbznbyygjzqnqlnaulszhvrngcdsamodyyxkajdbm
4
sjtlxykyjsdgjqqisaoftonewbhgaollosfgzampxrcrxefsdklswtugpjrfqyzxjtefikaybyonngbypuxwtytzzcbleuflmeli
6
oxcjewpnlnhdcuajwbaaaxdbxwcvvuemaagcatlvehxruxttcnhchoyuegktquasfrnqgswqqissxsfacyogihrvzrlverajzzxx
2
bvoqbfbeufzxettaemrheunmobspovzkbsjzlkbtszndfrsvvakfkwzdpfkdwbsmqclfskfwljkcuturwywxsamewnrcpbjchrlh
4
gkyldklmzlyxgfjohhxbprbdqruagekihqgbslxerwqbciyvzkxqxyzrgsiykibfrrlubbuqzmyznzdlapnsmagqnqnuwhkuljrh
5
xkxbnsdqghakektsrultbglzwnjmaihqqjzyrtptvrmmpyhqbckygtlyjpsoeblsydbrajzdkcezvwxjdqbsjfdhvonjllrcypwi
8
otyhlwiwkiybggdlxeycshyuxhsvmxcrbcctrzywbjnzqpjyaesakslxmggbkyyvncitvfzxdxhpokakzhysfsfeyogmawihnacg
6
