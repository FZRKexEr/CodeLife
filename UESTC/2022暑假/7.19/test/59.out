33554432
882499718
2048
33554432
65536
65980984
33554432
1048576
4096
65536
